resources/js/bootstrap.js

```diff
@@ -16,7 +16,12 @@ document.addEventListener("DOMContentLoaded", function () {
         require("./../../node_modules/bootstrap/js/dist/modal");
         require("./../../node_modules/bootstrap/js/dist/tab");
         require("./../../node_modules/bootstrap/js/dist/dropdown");
-        require("./../../node_modules/mdb-ui-kit/js/mdb.min");
+        window.bootstrap = require("./../../node_modules/mdb-ui-kit/js/mdb.min");
```

Comment:
I don’t know if this breaks modals and Dropdowns. But I think it does. Can‘t test rn.

Reply:
https://getbootstrap.com/docs/5.0/getting-started/download/#npm